src/pybind/mgr/prometheus/module.py

```diff
@@ -1781,8 +1779,7 @@ def setup_default_config(self, server_addr: str, server_port: int) -> None:
         self.set_uri(build_url(scheme='http', host=self.get_server_addr(),
                      port=server_port, path='/'))
 
-    def setup_tls_using_cephadm(self, server_addr: str, server_port: int) -> None:
```

Comment:
plz don't remove existing code, as it's used for cephadm

Reply:
addressed